app/config.go

```diff
@@ -76,6 +79,8 @@ func BuildConfig() {
 
 	Config.GitCommitTitle = envString("COMPOSER_MR_COMMIT_TITLE", Config.GitCommitTitle)
 
+	Config.MRTitlePrefix = envString("COMPOSER_MR_TITLE_PREFIX", "Composer update: ")
```

Comment:
This should be 
```go
Config.MRTitlePrefix = envString("COMPOSER_MR_TITLE_PREFIX", Config.MRTitlePrefix)
```

In addition to this, `cmd/root.go` should get a new flag withim the `func init()` to set the default for that config variable, maybe something like:
```go
rootCmd.Flags().StringVarP(&app.Config.MRTitlePrefix, "title-prefix", "p", "Composer update:", "The merge request title prefix")
```
There is also another change I'll comment on shortly.